src/main/java/me/lokka30/microlib/maths/QuickTimer.java

```diff
@@ -65,7 +65,7 @@ public void start() {
      *
      * @since 3.1.3
      */
-    public void startNano() {
+    public void startNanoTimer() {
         startTime = System.nanoTime();
     }
 
```

Comment:
I think most of this class needs to be re-done. Don't worry, it's a very easy task.

* In the constructor, it should specify what `TimerUnits` should be used for the QuickTimer (nano, milli). This is stored in a `private final` variable.
* There will be these methods
  * `reset()`
    * Sets the starting time to whenever it was ran
  * `getDuration()`
    * Gets how long it has been since the starting time
  * `getTimerUnits()`
    * Self explanatory

I will work on this unless you insist :)

Reply:
Well i'm busy until Friday GMT+1, gotta job to work. So, if you can do that i would be really gratefull. 😁

Reply:
All done in upcoming commit ;)